examples/scripts/minimal/ppo.py

```diff
@@ -0,0 +1,108 @@
+import shutil
+
+from datasets import load_dataset
+from transformers import (
+    AutoModelForCausalLM,
+    AutoModelForSequenceClassification,
+    AutoTokenizer,
+    HfArgumentParser,
+)
+
+from trl import ModelConfig
+from trl.trainer.ppov2_trainer import PPOConfig, PPOTrainer
+from trl.trainer.utils import SIMPLE_QUERY_CHAT_TEMPLATE
+
+
+"""
+python -i examples/scripts/minimal/ppo.py \
+    --learning_rate 3e-6 \
+    --output_dir models/minimal/ppo \
+    --per_device_train_batch_size 64 \
+    --gradient_accumulation_steps 1 \
+    --total_episodes 10000 \
+    --model_name_or_path EleutherAI/pythia-1b-deduped \
+    --non_eos_penalty \
+
+accelerate launch --config_file examples/accelerate_configs/deepspeed_zero3.yaml \
+    examples/scripts/minimal/ppo.py \
+    --output_dir models/minimal/ppo \
+    --num_ppo_epochs 1 \
+    --num_mini_batches 1 \
+    --learning_rate 3e-6 \
+    --per_device_train_batch_size 1 \
+    --gradient_accumulation_steps 16 \
+    --total_episodes 10000 \
+    --model_name_or_path EleutherAI/pythia-1b-deduped \
+    --sft_model_path EleutherAI/pythia-1b-deduped \
+    --reward_model_path EleutherAI/pythia-1b-deduped \
+    --local_rollout_forward_batch_size 1 \
+    --deepspeed3 \
+    --non_eos_penalty \
+"""
+
+
+if __name__ == "__main__":
+    parser = HfArgumentParser((PPOConfig, ModelConfig))
+    config, model_config = parser.parse_args_into_dataclasses()
+    # remove output_dir if exists
+    shutil.rmtree(config.output_dir, ignore_errors=True)
```

Comment:
FYI you can set `overwrite_output_dir` in `PPOConfig` (via `TrainingArguments`)

Reply:
I gave it a quick test but it does not seem to remove the output_dir. 

<img width="1741" alt="image" src="https://github.com/huggingface/trl/assets/5555347/2c7af03c-a246-43c1-9a5b-a3f8632649b3">

A quick search shows that the removing logic seems no longer there.

<img width="1383" alt="image" src="https://github.com/huggingface/trl/assets/5555347/a59e82e4-f316-45c0-a1df-fa80a17e3e20">